pvr.vuplus/changelog.txt

```diff
@@ -1,3 +1,6 @@
+v4.6.0
```

Comment:
And here

Reply:
I look after I updated all Leia Version, maybe we use direct 5.0.0 to prevent this?

Reply:
I'll leave @ksooo chime in here.

Reply:
Leia currently uses version 3.27.0, so why do you want to do a major bump to 5.0.0?

Reply:
Forget it for here, sorry seen only on other addons.